src/components/Headers/UserProfileHeader/UserProfileHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import GoBackButton from '@/components/GoBackButton/GoBackButton'
+import SocialItems from '@/components/SocialItems/SocialItems'
+import { UserProfileHeaderType } from '../types'
+
+import styles from './UserProfileHeader.module.scss'
+import classNames from 'classnames/bind'
+
+const cx = classNames.bind(styles)
+
+export default function UserProfileHeader({
+  userProfile,
+  withBackButton,
+}: UserProfileHeaderType) {
+  return (
+    <div
+      className={cx('wrapper', {
+        [styles.withBackBackButton]: !!withBackButton,
+      })}
+    >
```

Comment:
Take a look at the second image, doesn't it look clearer that way? We define the class names upfront and then apply a single class name to the JSX element 
![darek1](https://github.com/nerdbord/good-dev-hunting-app/assets/104648914/8c5945d7-43bc-4843-882f-600dcb6a5117)
![darek2](https://github.com/nerdbord/good-dev-hunting-app/assets/104648914/dada7e22-1dfd-497a-9f65-986da4ca4aea)